src/test/regress/sql/multi_extension.sql

```diff
@@ -24,60 +24,16 @@ WHERE pgd.refclassid = 'pg_extension'::regclass AND
 DROP EXTENSION citus;
 \c
 
--- Create extension in oldest version, test every upgrade step
+-- Create extension in oldest version, this is expected to fail
 CREATE EXTENSION citus VERSION '5.0';
-ALTER EXTENSION citus UPDATE TO '5.0-1';
-ALTER EXTENSION citus UPDATE TO '5.0-2';
-ALTER EXTENSION citus UPDATE TO '5.1-1';
-ALTER EXTENSION citus UPDATE TO '5.1-2';
-ALTER EXTENSION citus UPDATE TO '5.1-3';
-ALTER EXTENSION citus UPDATE TO '5.1-4';
-ALTER EXTENSION citus UPDATE TO '5.1-5';
-ALTER EXTENSION citus UPDATE TO '5.1-6';
-ALTER EXTENSION citus UPDATE TO '5.1-7';
-ALTER EXTENSION citus UPDATE TO '5.1-8';
-ALTER EXTENSION citus UPDATE TO '5.2-1';
-ALTER EXTENSION citus UPDATE TO '5.2-2';
-ALTER EXTENSION citus UPDATE TO '5.2-3';
-ALTER EXTENSION citus UPDATE TO '5.2-4';
-ALTER EXTENSION citus UPDATE TO '6.0-1';
-ALTER EXTENSION citus UPDATE TO '6.0-2';
-ALTER EXTENSION citus UPDATE TO '6.0-3';
-ALTER EXTENSION citus UPDATE TO '6.0-4';
-ALTER EXTENSION citus UPDATE TO '6.0-5';
-ALTER EXTENSION citus UPDATE TO '6.0-6';
-ALTER EXTENSION citus UPDATE TO '6.0-7';
-ALTER EXTENSION citus UPDATE TO '6.0-8';
-ALTER EXTENSION citus UPDATE TO '6.0-9';
-ALTER EXTENSION citus UPDATE TO '6.0-10';
-ALTER EXTENSION citus UPDATE TO '6.0-11';
-ALTER EXTENSION citus UPDATE TO '6.0-12';
-ALTER EXTENSION citus UPDATE TO '6.0-13';
-ALTER EXTENSION citus UPDATE TO '6.0-14';
-ALTER EXTENSION citus UPDATE TO '6.0-15';
-ALTER EXTENSION citus UPDATE TO '6.0-16';
-ALTER EXTENSION citus UPDATE TO '6.0-17';
-ALTER EXTENSION citus UPDATE TO '6.0-18';
-ALTER EXTENSION citus UPDATE TO '6.1-1';
-ALTER EXTENSION citus UPDATE TO '6.1-2';
-ALTER EXTENSION citus UPDATE TO '6.1-3';
-ALTER EXTENSION citus UPDATE TO '6.1-4';
-ALTER EXTENSION citus UPDATE TO '6.1-5';
-ALTER EXTENSION citus UPDATE TO '6.1-6';
-ALTER EXTENSION citus UPDATE TO '6.1-7';
-ALTER EXTENSION citus UPDATE TO '6.1-8';
-ALTER EXTENSION citus UPDATE TO '6.1-9';
-ALTER EXTENSION citus UPDATE TO '6.1-10';
-ALTER EXTENSION citus UPDATE TO '6.1-11';
-ALTER EXTENSION citus UPDATE TO '6.1-12';
-ALTER EXTENSION citus UPDATE TO '6.1-13';
-ALTER EXTENSION citus UPDATE TO '6.1-14';
-ALTER EXTENSION citus UPDATE TO '6.1-15';
-ALTER EXTENSION citus UPDATE TO '6.1-16';
-ALTER EXTENSION citus UPDATE TO '6.1-17';
-ALTER EXTENSION citus UPDATE TO '6.2-1';
+
+-- Create extension in newest major version, test every upgrade step
+CREATE EXTENSION citus VERSION '6.2-1';
 ALTER EXTENSION citus UPDATE TO '6.2-2';
 
+-- show running version
+SHOW citus.version;
+
 -- ensure no objects were created outside pg_catalog
 SELECT COUNT(*)
```

Comment:
Right after this `SELECT` statement, add a section that does a `RESET citus.enable_version_checks`, followed by a `DROP EXTENSION citus`, then a `CREATE EXTENSION citus VERSION '5.0';`. It should fail. The subsequent `DROP EXTENSION`/`CREATE EXTENSION` at the bottom of the file are sufficient as-is.